notebooks/en/benchmarking_tgi.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>"These parameters were changed if you're running on a Space because we don't want to conflict with the Spaces server:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>